CondTools/Geometry/plugins/PPSGeometryBuilder.cc

```diff
@@ -37,6 +37,7 @@ class PPSGeometryBuilder : public edm::one::EDAnalyzer<> {
 
   bool fromDD4hep_;
   std::string compactViewTag_;
+  bool isRun2_;
```

Comment:
@forthommel - out of curiosity why cannot you rely on an iov to determine that?

Reply:
@ianna, thanks for your post-review! Indeed an IOV could have worked too, but as there is only one algorithmic change (i.e. a few hardcoded values becoming "dynamic") a simple flag is probably more efficient. But this discussion is indeed quite dated (3-4 months), and I cannot fully remember the various arguments (if any).

Reply:
This was a long discussion. Eventually we decided to introduce this flag instead of re-uploading the Run2 geometry payloads.